src/authorization/WebAclAuthorizer.ts

```diff
@@ -184,20 +184,28 @@ export class WebAclAuthorizer extends Authorizer {
    *
    * @returns A store containing the relevant acl triples.
    */
-  private async getAclRecursive(id: ResourceIdentifier, recurse?: boolean): Promise<Store> {
+  private async getAclRecursive(id: ResourceIdentifier, maybeTarget?: ResourceIdentifier): Promise<Store> {
+    const target = maybeTarget ?? id;
```

Comment:
Can we give that second variable a more precise name though? Can it just be `target`?

Reply:
Was simply a "target" parameter in a first try, but then Jest kept complaining about untested 'branches' when I did not put the coalesced value in a constant first. I'll adapt the notation to something less functional though 😉